astro/src/content/docs/get-started/download-and-install/system-requirements.mdx

```diff
@@ -125,12 +125,19 @@ In most cases, if you have thousands of users and entities, `10GB` of disk space
 
 ## Network Access
 
-FusionAuth downloads the MySQL JDBC driver at install time. Due to the drivers' licensing, it cannot be bundled into the application. Prior to 1.16.0, the drivers were downloaded no matter which database you used. After that version, they are only downloaded if you are using MySQL.
+FusionAuth offers instructions on downloading the MySQL JDBC driver at install time. Due to the drivers' licensing, it cannot be bundled into the application. Prior to 1.16.0, the drivers were downloaded no matter which database you used. Prior to 1.40.0, the drivers were downloaded if you are using MySQL.
```

Comment:
This means that in `1.40.0` and later they must download and install the drivers even if the server has network access. Should we call that out?

Reply:
Good idea, done.